app/controllers/version.js

```diff
@@ -1,34 +1,33 @@
-import Controller from '@ember/controller';
+import Controller, {
+  inject as controller,
+} from '@ember/controller';
 import { inject as service } from '@ember/service';
+import { get, computed } from '@ember/object';
+
+function semverCompare (a, b) {
```

Comment:
May be we can use [semver-compare](https://www.npmjs.com/package/semver-compare) for this(we use it in [api docs](https://github.com/ember-learn/ember-api-docs/blob/70426b1d7815a23e2eab913104fc708f0c05d93e/app/utils/get-last-version.js#L4))